src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate.cs

```diff
@@ -458,6 +479,41 @@ public virtual string ToString(bool fVerbose)
             return sb.ToString();
         }
 
+        public virtual void Import(byte[] rawData)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
It's a cost:benefit at this point, as well as introducing a buggy area. It is not a scenario that is common and has been described as a "bug farm". It requires refactoring of the state that lives in the certificate (to move to PAL) and lots of tests to ensure that the objects can be re-imported and state gets properly cleared.